res/compile.py

```diff
@@ -0,0 +1,193 @@
+import os
+import subprocess
+import re
+import binascii
+import struct
+from multiprocessing.pool import ThreadPool
+
+# Set the path to the fxc compiler
+fxc = 'C:\\Program Files (x86)\\Microsoft DirectX SDK (June 2010)\\Utilities\\bin\\x86\\fxc.exe'
```

Comment:
No no, I'm just suggest to use DXSDK_DIR instead of absolute path, if we can use it of course.

Sorry, when I said:
> those backslashes **is** why?

I mean
> those backslashes, **that's** why we can't use DXSDK_DIR?